docs/dev_guide/diagnostic_ref/dpct1028.rst

```diff
@@ -27,3 +27,36 @@ Rewrite this code manually by using a supported random number generator. See the
 
 If the parameter is a CUDA device attribute:
 Redesign the code logic to remove use of the device attribute.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(int *result, CUdevice device) {
+      cuDeviceGetAttribute(result, CU_DEVICE_ATTRIBUTE_GPU_OVERLAP, device);
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "\*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done